javers-core/src/main/java/org/javers/core/graph/ObjectNode.java

```diff
@@ -75,7 +78,11 @@ public GlobalId getReference(Property property){
 
     public Object getPropertyValue(Property property) {
         Validate.argumentIsNotNull(property);
-        return cdo.getPropertyValue(property);
+        try {
+            return cdo.getPropertyValue(property);
+        } catch (JaversGetterException e){
```

Comment:
to nie jest połykanie tylko konwersja, nie mam 100% pewności, a masz jakiś pomysł co innego może się tu stać? bez tego, użytkownik dostaje IllegalArgumentException z zupełnie nic nie mówiącym komunikatem